build.sbt

```diff
@@ -21,17 +22,15 @@ libraryDependencies ++= Seq(
   Library.Test.logging
 )
 
+publishMavenStyle := true
 publishTo := {
-  val prefix               = if (isSnapshot.value) "snapshots" else "releases"
-  val s3BucketFriendlyName = "HAT Library Artifacts"
-  val s3BucketName         = "library-artifacts-"
-  val s3DomainSuffix       = ".hubofallthings.com"
+  val prefix = if (isSnapshot.value) "snapshots" else "releases"
   Some(
-    s3resolver
-      .value(List(s3BucketName, prefix).mkString(""), s3(s3BucketName + prefix + s3DomainSuffix)) withMavenPatterns
+    "Models" + prefix at "s3://library-artifacts-" + prefix + ".hubofallthings.com"
```

Comment:
I forgot to comment in another PR where the same pattern is used but we should use templated strings, the plus signs make this hard to read, and they rely on the not so intuitive rules of operator precedence

Reply:
will fix